packages/mwp-store/src/server/fetchQueries.js

```diff
@@ -1,21 +1,16 @@
 // @flow
 import 'rxjs/add/operator/map';
 import 'rxjs/add/operator/toPromise';
-import {
-	parseQueryResponse,
-	MEMBER_COOKIE_NAME,
-	getAuthedQueryFilter,
-} from '../util/fetchUtils';
+import { parseQueryResponse, getAuthedQueryFilter } from '../util/fetchUtils';
 
 /**
  * on the server, we can proxy the API requests directly without making a
  * request to the server's own API proxy endpoint
  */
-export default (request: HapiRequest) => () => (
+export default (request: HapiRequest) => (apiUrl, member) => (
```

Comment:
Not seeing where `apiUrl` is being used

Reply:
it's not used on the server, but it's part of the `fetchQueries` signature, so I've explicitly named it here